samtranslator/model/lambda_.py

```diff
@@ -152,6 +152,13 @@ class LambdaLayerVersion(Resource):
         "LicenseInfo": PropertyType(False, IS_STR),
     }
 
+    Content: Dict[str, Any]
+    Description: Optional[Intrinsicable[str]]
+    LayerName: Optional[Intrinsicable[str]]
+    CompatibleArchitectures: Optional[List[Union[str, Dict[str, Any]]]]
+    CompatibleRuntimes: Optional[List[Union[str, Dict[str, Any]]]]
+    LicenseInfo: Optional[Intrinsicable[str]]
```

Comment:
nit: any of these can be replaced with `PassThrough`?

Reply:
Good call.

Reply:
Actually no, this is `AWS::Lambda::LayerVersion`. None of the properties should be PassThrough. `PassThrough` should only be used on SAM resources right? @aahung